test/e2e-test/hack/addon-vela-test.sh

```diff
@@ -49,6 +49,22 @@ vela addon enable cert-manager
 vela addon enable flink-kubernetes-operator
 # set back to the DEFAULT_VELA_NS
 declare -x DEFAULT_VELA_NS=vela-system
+
+# test pyroscope addon
+# enable pyroscope
+kubectl create ns pyroscope-system
+declare -x DEFAULT_VELA_NS=pyroscope-system
+vela addon enable fluxcd
+vela addon enable pyroscope
+declare -x DEFAULT_VELA_NS=vela-system
+
+# disable pyroscope
+declare -x DEFAULT_VELA_NS=pyroscope-system
+vela addon disable pyroscope
```

Comment:
Is this addon should be installed `pyroscope-system` namespace

Reply:
I refer to fluxcd-system，and a separate ns is more clear to user？

Reply:
It is default to `vela-system`